test/unit/messaging/messaging.spec.ts

```diff
@@ -2852,6 +2860,9 @@ describe('Messaging', () => {
               bodyLocArgs: ['arg1', 'arg2'],
               channelId: 'test.channel',
             },
+            fcmOptions: {
+              analyticsLabel: 'test.analytics',
```

Comment:
Can you also add some test cases to make sure the new validators work as intended?

Reply:
OK done